packages/maskbook/src/social-network-provider/twitter.com/ui/injectPostInspector.tsx

```diff
@@ -10,7 +10,7 @@ export function injectPostInspectorAtTwitter(current: PostInfo) {
             if (content) {
                 for (const a of content.querySelectorAll('a')) {
                     if (twitterEncoding.payloadDecoder(a.title)) hideDOM(a)
-                    if (a.title === 'http://maskbook.com') hideDOM(a)
+                    if (a.title === 'http://mask.io' || a.title === 'http://maskbook.com') hideDOM(a)
```

Comment:
```suggestion
                    if (/^https?:\/\/mask(\.io|book\.com)$/i.test(a.title)) hideDOM(a)
```

Reply:
No, I don't find any good reason to prefer regexp. It's harder to read and harder to maintain. If you're not handling a very flexible string match, you don't need it.